x/leverage/simulation/operations_test.go

```diff
@@ -235,24 +230,45 @@ func (s *SimTestSuite) TestSimulateMsgBorrowAsset() {
 	s.Require().Len(futureOperations, 0)
 }
 
-func (s *SimTestSuite) TestSimulateMsgSetCollateralSetting() {
+func (s *SimTestSuite) TestSimulateMsgAddCollateral() {
 	r := rand.New(rand.NewSource(1))
 
 	accs := s.getTestingAccounts(r, 3, func(fundedAccount simtypes.Account) {})
 
 	s.app.BeginBlock(abci.RequestBeginBlock{Header: tmproto.Header{Height: s.app.LastBlockHeight() + 1, AppHash: s.app.LastCommitID().Hash}})
 
-	op := simulation.SimulateMsgSetCollateralSetting(s.app.AccountKeeper, s.app.BankKeeper, s.app.LeverageKeeper)
+	op := simulation.SimulateMsgAddCollateral(s.app.AccountKeeper, s.app.BankKeeper, s.app.LeverageKeeper)
 	operationMsg, futureOperations, err := op(r, s.app.BaseApp, s.ctx, accs, "")
 	s.Require().NoError(err)
 
-	var msg types.MsgSetCollateral
+	var msg types.MsgAddCollateral
 	types.ModuleCdc.UnmarshalJSON(operationMsg.Msg, &msg)
 
 	s.Require().True(operationMsg.OK)
 	s.Require().Equal("umee1ghekyjucln7y67ntx7cf27m9dpuxxemn8w6h33", msg.Borrower)
-	s.Require().Equal(types.EventTypeSetCollateralSetting, msg.Type())
-	s.Require().Equal(true, msg.Enable)
+	s.Require().Equal(types.EventTypeAddCollateral, msg.Type())
+	s.Require().Equal("0u/uabc", msg.Amount.String())
+	s.Require().Len(futureOperations, 0)
+}
+
+func (s *SimTestSuite) TestSimulateMsgRemoveCollateral() {
+	r := rand.New(rand.NewSource(1))
+
+	accs := s.getTestingAccounts(r, 3, func(fundedAccount simtypes.Account) {})
+
+	s.app.BeginBlock(abci.RequestBeginBlock{Header: tmproto.Header{Height: s.app.LastBlockHeight() + 1, AppHash: s.app.LastCommitID().Hash}})
+
+	op := simulation.SimulateMsgRemoveCollateral(s.app.AccountKeeper, s.app.BankKeeper, s.app.LeverageKeeper)
+	operationMsg, futureOperations, err := op(r, s.app.BaseApp, s.ctx, accs, "")
+	s.Require().NoError(err)
+
+	var msg types.MsgRemoveCollateral
+	types.ModuleCdc.UnmarshalJSON(operationMsg.Msg, &msg)
+
+	s.Require().True(operationMsg.OK)
+	s.Require().Equal("umee1ghekyjucln7y67ntx7cf27m9dpuxxemn8w6h33", msg.Borrower)
+	s.Require().Equal(types.EventTypeRemoveCollateral, msg.Type())
+	s.Require().Equal("0u/uabc", msg.Amount.String())
 	s.Require().Len(futureOperations, 0)
```

Comment:
suggestion, let's declare `require := s.Require()` at the top, and then use `require.Equal(...)` instead of `s.Require().Equal(...)` - it simplifies reading.

Reply:
I'll make a note to do that in all the tests, in another PR